routes/alfresco.py

```diff
@@ -538,21 +671,31 @@ def run_fetch_alf_local_data(var_ep, lat, lon):
             # otherwise take nearest HUC within 100m
             huc_id = huc12_gdf.iloc[idx_arr[np.argmin(near_distances)]].name
 
-    huc12_pkg = run_fetch_alf_area_data(var_ep, huc_id)
+    huc12_pkg = run_fetch_alf_area_data(var_ep, huc_id, ignore_csv=True)
+
+    if request.args.get("format") == "csv":
+        huc12_pkg = nullify_and_prune(huc12_pkg, "alfresco")
+        if huc12_pkg in [{}, None, 0]:
+            return render_template("404/no_data.html"), 404
+        # return CSV with lat/lon info since HUC12 names not handled
+        return create_csv(huc12_pkg, var_ep, huc_id, lat=lat, lon=lon)
+
     huc12_pkg["huc_id"] = huc_id
     huc12_pkg["boundary_url"] = f"https://earthmaps.io/boundary/area/{huc_id}"
 
     return huc12_pkg
 
 
 @routes.route("/alfresco/<var_ep>/area/<var_id>")
-def run_fetch_alf_area_data(var_ep, var_id):
+def run_fetch_alf_area_data(var_ep, var_id, ignore_csv=False):
```

Comment:
Added this to handle HUC12 local CSVs. If you think of a strategy you like more, please go for it!